### DISCUSSION TITLE
Ubuntu 24.04 LTS update any time soon?

### DISCUSSION THREAD

**Discussion by ricperry** [TheAssassin/AppImageLauncher#642] [Q&A] [answered]
### Pre-submit checks

- [X] I checked for similar issues beforehand, but could not find any. I could not add my proposal to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Feature description

I just want to be able to "install" appimages to the appropriate location and added to the launcher menus without the hassle of manually placing and editing files myself. Is this going to be coming to Ubuntu 24.04 LTS any time soon?

### Proposed solution

Package the build with required dependencies for Gnome 46 / Ubuntu 24.04 LTS

### Alternative solutions you considered

I'm not aware of any other than laboriously installing individual appimages myself.

**Comment by InklingGirl** [comment]
+1

**Comment by erkinalp** [comment]
depends on #634

**Comment by Munk3y** [comment]
+1

**Comment by TheAssassin** [comment, marked as answer]
The `.deb` from the [release page](https://github.com/TheAssassin/AppImageLauncher/releases/tag/continuous) works fine on Ubuntuu 24.04. I use it every day, on multiple systems.

Before you ask: the PPA is broken and I don't think a fix is on the way. That's tracked in a separate issue.

**Reply by erkinalp** [reply]
i tried that one, the "integrate me" dialog appears, but appimages themselves fail to load

**Reply by TheAssassin** [reply]
Have you installed libfuse2? https://docs.appimage.org/user-guide/troubleshooting/fuse.html

**Reply by erkinalp** [reply]
That fixed the issue, thanks.

**Reply by vvuk** [reply]
Just to confirm -- we're talking about the .deb from _2022_? That's great if so (and I'm installing it now), but my first thought at coming across this project was "oh wow 2022, this looks abandoned".  Likewise the README only mentions up to Ubuntu 19/Debian 10. Maybe worth just updating the README to mention some of the newer versions to make it clear that the last release is actually the current up-to-date and still-working one?

**Comment by ricperry** [comment]
Thanks. I wasn't aware of the release page, and was just trying to follow the instructions from the main page of this git.
![Screenshot from 2024-05-06 12-57-25](https://github.com/TheAssassin/AppImageLauncher/assets/34995372/3820e9ad-4a1f-4f5d-9d2d-da8ec2ca91f3)

**Comment by TheAssassin** [comment]
The README is a little out of date. I still have hopes for the PPA, but right now, it's broken.

**Comment by InklingGirl** [comment]
:(

**Comment by shawe** [comment]
https://docs.appimage.org/user-guide/troubleshooting/fuse.html#setting-up-fuse-2-x-alongside-of-fuse-3-x-on-recent-ubuntu-22-04-debian-and-their-derivatives

For Ubuntu 24.04 is actually the package libfuse2t64 that must be installed, but for me the problem appears some days ago.

What exactly info can I provide you to help debug with this problem?

Launching in terminal I see that:

```
$ ./PrusaSlicer-2.8.0+linux-x64-GTK3-202406271021-Ubuntu-24-04_8faf233db5caf189ea8dd8b57f269866.AppImage 
AppImageLauncher error: appimage_shall_not_be_integrated() failed (returned -1)
AppImageLauncher error: appimage_is_terminal_app() failed (returned -1)
execv error: No such file or directory
```

**Reply by TheAssassin** [reply]
Yes, indeed you need to install `libfuse2` on Ubuntu 24.04 for most current AppImages. This is going to change sooner than later.

Please just open an issue, the issue template will request all the data accordingly.

**Reply by shawe** [reply]
How you can see, now its libfuse2t64 on Ubuntu 24.04, how I advice you before:

```
$ sudo apt-get install libfuse2
[sudo] contraseña para francesc: 
Leyendo lista de paquetes... Hecho
Creando árbol de dependencias... Hecho
Leyendo la información de estado... Hecho
Nota, seleccionando «libfuse2t64» en lugar de «libfuse2»
libfuse2t64 ya está en su versión más reciente (2.9.9-8.1build1).
0 actualizados, 0 nuevos se instalarán, 0 para eliminar y 0 no actualizados.
```

Thanks to the snapshots of PrusaSlicer, last time that was working fine was in 20240727T174306Z. From that day, only system updates installed, no other changes. And AppImageLauncher now is not working.

Switching to flatpak version of PrusaSlicer until AppImage works again.

**Reply by TheAssassin** [reply]
Interesting.

For the record, I'm using AppImageLauncher (installed from the repo-provided `.deb`) every day with just that version of PrusaSlicer on Ubuntu 24.04 and Debian 12 myself. I have not had this specific issue before.

But anyway, please open an issue and fill in all the fields.

**Reply by shawe** [reply]
No way to debug what is happening to my case?

**Reply by TheAssassin** [reply]
This is the last time I'll be asking you to ***open an issue*** instead of continuing this discussion. There is a template. It asks you the required data. I'm not going to repeat that here because you refuse to fill in a simple form.

**Reply by TheAssassin** [reply]
https://github.com/TheAssassin/AppImageLauncher/issues/new/choose

**Comment by LiamSaboridoSueiro** [comment]


**Reply by AntonOfTheWoods** [reply]
Ok, maybe I'm missing something. Is this a spam comment? The `AirSim` repo doesn't seem to have appimages and they also clearly state on the `readme` - "Users will still have access to the original AirSim code beyond that point, but no further updates will be made, effective immediately. ", and that was almost 2 years ago...

**Reply by LiamSaboridoSueiro** [reply]
Hey, sorry for the confusion—I misread the discussion. I initially thought they were asking for an updated version of AirSim. I had to update it to work with that specific version of Ubuntu, so I figured it might be helpful to share what I had discovered.
